Lab2.ipynb

```diff
@@ -0,0 +1,830 @@
+      "text/plain": [
+       "<Figure size 720x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (10,7)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "x = np.linspace(-5,5)\n",
+    "background = stats.norm.pdf(x,loc = 0, scale = 1.5)\n",
+    "ax.plot(x, background, label='norm pdf');\n",
+    "plt.xlabel('brightness', fontsize=18)\n",
+    "plt.ylabel('probability', fontsize=18);\n",
+    "\n",
+    "signal_strength = 4.2\n",
+    "prob = stats.norm.pdf(signal_strength,loc = 0, scale = 1.5)\n",
```

Comment:
I think `stats.norm.pdf` gives the probability density function. We should use `1-cdf()` or `sf()` to calculate the probability that the data produces a signal that is equally or more signal-like than the observation.